[0.063s] Invoking command in '/home/pi2/ros2_ws/build/my_parameters': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/opt/ros/humble /usr/bin/cmake /home/pi2/ros2_ws/src/my_parameters -DCMAKE_INSTALL_PREFIX=/home/pi2/ros2_ws/install/my_parameters
[0.112s] -- Found ament_cmake: 1.3.10 (/opt/ros/humble/share/ament_cmake/cmake)
[0.777s] -- Found ament_lint_auto: 0.12.11 (/opt/ros/humble/share/ament_lint_auto/cmake)
[1.120s] -- Found geometry_msgs: 4.2.4 (/opt/ros/humble/share/geometry_msgs/cmake)
[1.295s] -- Found rosidl_generator_c: 3.1.5 (/opt/ros/humble/share/rosidl_generator_c/cmake)
[1.308s] -- Found rosidl_adapter: 3.1.5 (/opt/ros/humble/share/rosidl_adapter/cmake)
[1.335s] -- Found rosidl_generator_cpp: 3.1.5 (/opt/ros/humble/share/rosidl_generator_cpp/cmake)
[1.381s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[1.532s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[1.819s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[2.757s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[2.758s]   execute_process(/usr/bin/python3 -m rosidl_adapter --package-name
[2.758s]   my_parameters --arguments-file
[2.759s]   /home/pi2/ros2_ws/build/my_parameters/rosidl_adapter__arguments__my_parameters.json
[2.759s]   --output-dir
[2.760s]   /home/pi2/ros2_ws/build/my_parameters/rosidl_adapter/my_parameters
[2.760s]   --output-file
[2.761s]   /home/pi2/ros2_ws/build/my_parameters/rosidl_adapter/my_parameters.idls)
[2.761s]   returned error code 1:
[2.761s] 
[2.762s]   Traceback (most recent call last):
[2.762s] 
[2.763s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[2.763s]       return _run_code(code, main_globals, None,
[2.764s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[2.765s]       exec(code, run_globals)
[2.766s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[2.767s]       sys.exit(main())
[2.767s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[2.768s]       abs_idl_file = convert_to_idl(
[2.769s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 19, in convert_to_idl
[2.771s]       return convert_msg_to_idl(
[2.771s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/msg/__init__.py", line 28, in convert_msg_to_idl
[2.772s]       msg = parse_message_string(package_name, input_file.stem, content)
[2.773s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 544, in parse_message_string
[2.774s]       msg = MessageSpecification(pkg_name, msg_name, fields, constants)
[2.775s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 388, in __init__
[2.776s]       self.base_type = BaseType(
[2.777s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 205, in __init__
[2.778s]       raise InvalidResourceName(
[2.779s] 
[2.779s]   rosidl_adapter.parser.InvalidResourceName: 'motor_parameters' is an invalid
[2.780s]   message name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$'
[2.780s] 
[2.780s] Call Stack (most recent call first):
[2.781s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[2.781s]   CMakeLists.txt:29 (rosidl_generate_interfaces)
[2.782s] 
[2.782s] [0m
[2.783s] -- Configuring incomplete, errors occurred!
[2.783s] See also "/home/pi2/ros2_ws/build/my_parameters/CMakeFiles/CMakeOutput.log".
[2.786s] Invoked command in '/home/pi2/ros2_ws/build/my_parameters' returned '1': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/opt/ros/humble /usr/bin/cmake /home/pi2/ros2_ws/src/my_parameters -DCMAKE_INSTALL_PREFIX=/home/pi2/ros2_ws/install/my_parameters
